client/MentorPanel/src/components/input.js

```diff
@@ -65,21 +65,13 @@ const InputField = ({ text, onSelect, onChange }) => {
 class Input extends React.Component {
   constructor(props) {
     super(props);
-    this.state = { text: '', height: 0 };
+    this.state = { text: '' };
```

Comment:
let's see if we can get this to work with `useState` hook and a functional component as discussed